pykern/pkunit.py

```diff
@@ -222,6 +185,76 @@ def file_eq(expect_path, *args, **kwargs):
     import pykern.pkjson
     import pykern.pkconst
 
+    def _convert_xlsx(expect_path, actual_path):
+        if expect_path.ext == '.csv' and not actual_path.check(file=True):
```

Comment:
Hey @robnagler what is the point of the ```not actual_path.check(file=True)``` part of the conditional check?

If the expected is .csv and the .xlsx does not exist, then what is the purpose of converting? 
And if the .xlsx does exist don't we want to compare the conversion to the expected .csv?